### PR TITLE
drivers: wifi: Display firmware version

### PR DESCRIPTION
**Pull request by rado17**
Add command in wifi_util shell to display firmware version.

### CODE REVIEW

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c [hunk 1]
```diff
@@ -491,6 +491,35 @@ static int nrf_wifi_util_tx_rate(const struct shell *shell,
...
+	struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx = NULL;
+	unsigned int umac_ver, lmac_ver;
+
+	fmac_dev_ctx = ctx->rpu_ctx;
+
+	status = wifi_nrf_fmac_ver_get(fmac_dev_ctx, &umac_ver, &lmac_ver);
+
+	if (status == WIFI_NRF_STATUS_SUCCESS) {
+		shell_fprintf(shell, SHELL_INFO,
+		 "UMAC : %d.%d.%d.%d\nLMAC : %d.%d.%d.%d\n",
```
Comment (sachinthegreen) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c:508]: `UMAC version` and `LMAC version`

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c (hunk 1 above)
Comment (rado17) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c]: Done

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c [hunk 2]
```diff
@@ -491,6 +491,35 @@ static int nrf_wifi_util_tx_rate(const struct shell *shell,
...
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx = NULL;
+	unsigned int umac_ver, lmac_ver;
+
+	fmac_dev_ctx = ctx->rpu_ctx;
+
+	status = wifi_nrf_fmac_ver_get(fmac_dev_ctx, &umac_ver, &lmac_ver);
+
+	if (status == WIFI_NRF_STATUS_SUCCESS) {
```
[the remark this answers is not in the document]
Comment (rado17) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c]: Done

**Review by sachinthegreen — APPROVED** [review]

**Review by krish2718 — APPROVED** [review]

**Comment by rlubos** [comment]
@rado17 Please fix the compliance issue:
> drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c:510  Statements should start on a tabstop

**Pull request merged**